habitat_baselines/config/pointnav/ppo.yaml

```diff
@@ -0,0 +1,40 @@
+TRAINER:
+  TRAINER_NAME: "ppo"
+  RL:
+    PPO:
+      # general options
+      tensorboard_dir: "tb"
+      num_processes: 1
+      log_interval: 10
+      pth_gpu_id: 0
+      sim_gpu_id: 0
+      checkpoint_interval: 50
+      checkpoint_folder: "data/new_checkpoints"
+      task_config: "configs/tasks/pointnav.yaml"
+      num_updates: 10000
+      # eval specific:
+      count_test_episodes: 2
+      video_option: ["disk", "tensorboard"]
+      video_dir: "video_dir"
+      model_path: "data/checkpoints/ckpt.123.pth"
+      tracking_model_dir: ""
+      # 1 of the above 2 options needs to be empty
```

Comment:
Is there an explicit check for this somewhere.

Reply:
I did a check [here](https://github.com/JasonJiazhiZhang/habitat-api/blob/c672710abfa3c58cce086ff623afb92596f7f47f/habitat_baselines/rl/ppo/ppo_trainer.py#L291-L293).